src/Main.hs

```diff
@@ -0,0 +1,20 @@
+import System.Environment (getArgs)
+
+import Month (monthString)
+import Year (yearString)
+
+main :: IO ()
+main = do
+  args <- map read <$> getArgs -- TODO: read will explode on non-numeric input
```

Comment:
Why do you need to map read over/into an fmap of getArgs?
My guess is fmap lifts the values from get args into a regular list that can then just be mapped over?